code/modules/cargo/packs.dm

```diff
@@ -285,6 +285,18 @@
 					/obj/item/ammo_box/c38/iceblox)
 	crate_name = "ammo crate"
 
+/datum/supply_pack/security/nemesis
+	name = "Nemesis Solutions Equipment"
+	desc = "Contains full equipment for one security officer and a high-tech combat vest from Nemesis Solutions along with some gadgets for them to use."
+	cost = CARGO_CRATE_VALUE * 4
```

Comment:
Hoooookay no.
Jack up the price, existing security equipment is already relatively cheap, but this is literally purchasable by heads and high paying roles roundstart.